promql/engine.go

```diff
@@ -1180,8 +1180,7 @@ func (ev *evaluator) eval(expr parser.Expr) (parser.Value, storage.Warnings) {
 		}
 
 		unwrapParenExpr(&e.Param)
-		param := unwrapStepInvariantExpr(e.Param)
-		unwrapParenExpr(&param)
+		param := unwrapStepInvariantExprAndParenExpr(e.Param)
```

Comment:
Another suggestion: Let's also have this first `unwrapParenExpr` inside `unwrapStepInvariantExprAndParenExpr`

Reply:
Hence a single call to unwrap it all together.

Reply:
changed